docs/support_playbook.md

```diff
@@ -335,23 +335,12 @@ If the course details have changed from one cycle to another, provider users sho
 application_choice = ApplicationChoice.find(APPLICATION_CHOICE_ID)
 new_course_option = CourseOption.find(NEW_COURSE_OPTION_ID)
 zendesk_url = ZENDESK_URL
+support_user = SupportUser.find_by(email_address: YOUR_SUPPORT_EMAIL)
 
-# confirm the deferral in a new course
-application_choice.update_course_option_and_associated_fields!(new_course_option, audit_comment: zendesk_url)
-```
-
-A **conditional offer** would move the candidate to a pending conditions state:
+application_choice.audit_comment = zendesk_url
```

Comment:
I'm not sure if this will work. I think it might get overwritten in the `update_course_option_and_associated_fields!` method which takes an `audit_comment`. Have you tested it locally?

Reply:
Yeah I tested locally and it was added to one of the Audit changes - there were 2 audits created one changing the status (which has the comment attached) and the other was changing the course option, which didn't get the audit comment. 

I'd be happy with this "for now"™️ since we're looking at implementing this functionality for Providers 🤞